test/Makefile

```diff
@@ -129,6 +129,7 @@ CFLAGS ?= --target=$(TARGET_TRIPLE) --sysroot=$(SYSROOT_DIR)
 # Always include the `libc-test` infrastructure headers.
 CFLAGS += -I$(INFRA_HEADERS_DIR)
 ifneq ($(findstring -threads,$(TARGET_TRIPLE)),)
+LDFLAGS += -Wl,--import-memory,--export-memory,--shared-memory,--max-memory=1073741824
```

Comment:
Did you find that doing this same thing via the `//! add-flags.py(LDFLAGS): ...` annotation in the test files didn't work?

Reply:
Yes, it should work but I did it in this way as I think those flags are not a property of test cases but a property of the compiler target. Same with the engine flags.

Reply:
I see what you mean. Practically, though, it might be more convenient to move this to the test files since there may be discrepancies between targets; e.g., what if other threads tests don't require setting up 1G of max memory?

Reply:
And, honestly, the simpler we can make this `Makefile`, the better.

Reply:
Ok, it makes sense to me 👍